types/react-native-canvas/index.d.ts

```diff
@@ -1,21 +1,45 @@
 // Type definitions for react-native-canvas 0.1
 // Project: https://github.com/iddan/react-native-canvas#readme
 // Definitions by: hmajid2301 <https://github.com/hmajid2301>
+//                 Kelvin Chu <https://github.com/ragebill>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.5
```

Comment:
Why was the TS version upgrade needed?

Reply:
The main reason is because I personally used and tested this on Typescript 3.5. Although this change is not needed, I tend to state the latest version of software that this definition has been tested on since people tend to adopt newer version of Typescript in their projects. Do you think it would be better to keep it as 3.1?

Reply:
Yeah this creates a lot of confusion, and yours was a reasonable way to interpret it. But this version states the _minimum required_ TypeScript version to be able to use these typings - so you'd only want to increment it if you're using newer TypeScript features in your typings, or depending on other packages that do. Keeping it on 3.1 means users on older versions of TS don't miss out on your changes.

Reply:
Didn't get my head around to realize it was the minimum required version! Thanks for the clarification here :)